src/plugins/auto/src/auto_compiled_model.cpp

```diff
@@ -109,81 +109,92 @@ ov::Any AutoCompiledModel::get_property(const std::string& name) const {
             real = m_scheduler->m_compile_context[ACTUALDEVICE].
                 m_compiled_model->get_property(name).as<unsigned int>();
         } else {
-            OPENVINO_ASSERT(m_scheduler->m_compile_context[CPU].m_is_already == true);
             std::unique_lock<std::mutex> lock(m_context->m_mutex);
             auto device_info = m_scheduler->m_compile_context[ACTUALDEVICE].m_device_info;
             lock.unlock();
             unsigned int optimal_batch_size = 0;
             unsigned int requests = 0;
             bool tput_enabled_in_plugin = false;
-            auto actual_dev_supported_properties = m_context->m_ov_core->get_property(device_info.device_name, ov::supported_properties);
-            try {
-                // for benchmark through AUTO:CPU,GPU
-                // SetConfig directly set to CPU/GPU in this case
-                if (std::find(actual_dev_supported_properties.begin(), actual_dev_supported_properties.end(), ov::hint::performance_mode)
-                    != actual_dev_supported_properties.end())
-                    tput_enabled_in_plugin =
-                        m_context->m_ov_core->get_property(device_info.device_name,
-                            ov::hint::performance_mode)== ov::hint::PerformanceMode::THROUGHPUT;
-            } catch (const ov::Exception&) {
-                LOG_DEBUG_TAG("get_property:%s for %s", "PERF_HINT config not supported",
-                    device_info.device_name.c_str());
+            auto actual_dev_supported_properties =
+                m_context->m_ov_core->get_property(device_info.device_name, ov::supported_properties);
+            auto ireqIter = std::find(actual_dev_supported_properties.begin(), actual_dev_supported_properties.end(), name);
+            if (ireqIter != actual_dev_supported_properties.end()) {
+                real = m_context->m_ov_core->get_property(device_info.device_name, ov::optimal_number_of_infer_requests);
```

Comment:
are we sure that this is the vpux plugin supported metric? in cpu/gpu, this seems to be a compiled model metric.
also, how do we place this ov::optimal_number_of_infer_requests against ov::num_request?

Reply:
1. ov::num_request pass to all devices in CTPUT hint - will have multiple times optimal_number_of_infer_requests
2. Without CPU as acceleration, get from compiled model
3. With CPU as acceleration
a. Get from plugin
pass all configurations when getting optimal_number_of_infer_requests from HW device
https://docs.openvino.ai/2022.3/classov_1_1Core.html#doxid-classov-1-1-core-1a3267437192896805958f2d6ccfd66429
b. Estimate by AUTO
min(ov::num_request, optimal_number_of_infer_requests estimated  by AUTO)

Reply:
Updated.

Reply:
Updated.